### DISCUSSION TITLE
How to force start of task

### DISCUSSION THREAD

**Discussion by acappelletti** [cybertec-postgresql/pg_timetable#528] [Q&A] [answered]
I created a chain-task (1-1) scheduled at a certain time. How can I force start now via psql?

Thanks Allen

**Comment by pashagolub** [comment, marked as answer]
Hello,

assuming your worker has name `worker01` and the chain name is `foojob` with ID `12345` you can execute this query to send a signal to the worker to start the execution:

```sql
SELECT timetable.notify_chain_start(12345, 'worker01'); -- start chain foojob with ID 12345 on a client worker01
```
You can also stop a running chain with this query:
```sql
SELECT timetable.notify_chain_stop(12345, 'worker01');
```
Kind regards

**Reply by acappelletti** [reply]
Thank you very much.
ALEN
